features/mbedtls/VERSION.txt

```diff
@@ -1,2 +1,2 @@
-mbedtls-2.15.0
-mbedcrypto-0.1.0b
+mbedtls-2.15.1
+mbedcrypto-0.1.0b2
```

Comment:
This has me a bit worried, but fine for now.

Reply:
What's worrying?

Reply:
Why worried? That's like half a comment - it's wrong but you're not saying how. :)

Reply:
Fair feedback. 

>  mbedcrypto-0.1.0b2 

This gives mes the impression that another TLS update will be needed before the feature release is complete. Is that correct?